src/controllers/CommentController.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+namespace App\Controllers;
+
+use App\Exceptions\CommentNotSavedException;
+use App\Models\Comment;
+
+class CommentController
+{
+
+    protected $commentModel;
+
+    public function __construct($config)
```

Comment:
The Law of Demeter would tell us that we shouldn't tell the object things it doesn't need to know, which include the config array. Consider removing in the next iteration when you inject the model itself.